content/en/blog/_posts/2021-XX-XX-writing-a-controller-for-pod-labels.md

```diff
@@ -0,0 +1,473 @@
+---
+layout: blog
+title: "Writing a Controller for Pod Labels"
+date: 2021-03-XX
+slug: writing-a-controller-for-pod-labels
+---
+
+**Authors**: Arthur Busser, Padok
+
+[Operators][what-is-an-operator] are proving to be an excellent solution to
+running stateful distributed applications in Kubernetes. Open Source tools like
+the [Operator SDK][operator-sdk] provide ways to build reliable and maintainable
+operators, and extending Kubernetes to implement custom scheduling needs is now
+easier than ever.
+
+Kubernetes operators run complex software inside your cluster. The open-source
```

Comment:
Use "open source" or "open-source" consistently throughout. This blog seems to prefer "open source" (214 instances) over "open-source" (57 instances).

Reply:
I've opted for "open source" 👍